tf_agents/environments/gym_wrapper.py

```diff
@@ -165,6 +165,11 @@ def __init__(self,
     self._action_spec = spec_from_gym_space(self._gym_env.action_space,
                                             spec_dtype_map, simplify_box_bounds,
                                             'action')
+    self._reward_spec = specs.BoundedArraySpec(shape=(),
```

Comment:
Is the gym_env.reward always np.float64 scalar? Or can it be float32 and/or a vector?